community/kernel/src/main/java/org/neo4j/storageengine/api/StoreReadLayer.java

```diff
@@ -78,6 +78,11 @@
     Iterator<NewIndexDescriptor> uniquenessIndexesGetAll();
 
     /**
+     * Returns all indexes (including unique) related to a property.
+     */
+    Iterator<NewIndexDescriptor> indexesAndUniqueIndexesRelatedToProperty( int propertyId );
```

Comment:
This will be done by the PR that is merge last of this and #9119. The reason this method was named like this is that for similar methods in the interface unique and general indexes are separated. That is changed in #9119.